README.md

```diff
@@ -1,10 +1,75 @@
 # mender-stress-test-client
-Mender stress test client
 
-This mender client uses the existing mender client package to create many "fake" mender clients using Golang go routines.
-This client successfully performs client authentication, and uses the backend API correctly, in order to emulate a device.
+The *mender-stress-test-client* is a dummy-client intended for simple testing of
+the Mender server functionality, or scale testing. It is not in any way a fully
+featured client. All it does is mimic a client's API, and download the update to
+/dev/null. This means that it has **no state** whatsoever, all updates are
+instantly thrown away, and the client will forever function as it did once
+booted up. This means that all parameters provided to the client at startup will
+stay this way until the client is brought down.
 
-To run with default options:
-`~/go/bin/mender-dummy -count <device count>`
+To run with the default options: `~/go/bin/mender-dummy -count <device count>`
```

Comment:
How to build it first?

Also, the binary will be `mender-stress-test-client` and I don't think it is relevant to have the `~/go/bin` prefix

Reply:
Lol, fair point.